uc-crux-llvm/src/UCCrux/LLVM/Context/Function.hs

```diff
@@ -117,24 +120,51 @@ ppFunctionContextError =
         ]
     BadMemType _ -> "Bad MemType"
 
+throwFunctionContextError :: FunctionContextError -> a
+throwFunctionContextError err =
+  case err of
+    BadMemType {} -> isMalformed
+    BadLift {} -> isMalformed
+    BadLiftArgs {} -> doPanic
+  where
+    nm = "throwFunctionContextError"
+    doPanic = panic nm [Text.unpack (ppFunctionContextError err)]
```

Comment:
If you do add `HasCallStack` constraints, these intermediate bindings could be problematic (unless they get their own `HasCallStack` constraints